mobile-core.addon

```diff
@@ -1,4 +1,4 @@
-# Name: mobilecore                                                                          
+# Name: minishift-mobilecore-addon                                                                          
```

Comment:
Don't we need to edit the `enable` and `apply` command in the doc to match this name, then?

Reply:
ah, yes. Will fix it